LogicAppsCustomCodeFramework/.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
i couldnt find a way to put this comment at the right place so putting it here - can we rename the root folder to indicate that this is a sample (not a framework) - e.g., LogicAppCustomCodeSample